### PR TITLE
[Images] Custom ID

### PR DESCRIPTION
**Pull request by marciocloudflare**

### CODE REVIEW

File: content/images/cloudflare-images/upload-images/custom-id.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+- Could include any number of subpaths.
+- Requires file extensions. For example `.png` or `.jpg`.
+- Supports the [UTF-8 encoding standard](https://en.wikipedia.org/wiki/UTF-8) for characters.
+
+Below is an example of the custom ID feature:
+
+```bash
+curl -X POST https://api.cloudflare.com/client/v4/accounts/<​​ACCOUNT_ID>/images/v1 \
+  -H "Authorization: Bearer <API_TOKEN>" \
+  --form 'url=https://<REMOTE_PATH_TO_IMAGE>' \
```
Comment (sejoker) [inline content/images/cloudflare-images/upload-images/custom-id.md:21]: let's use consistent notation
```suggestion
  -F 'url=https://<REMOTE_PATH_TO_IMAGE>' \
```

File: content/images/cloudflare-images/upload-images/custom-id.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  --form 'id=<PATH_TO_YOUR_IMAGE.IMG>'
```
Comment (sejoker) [inline content/images/cloudflare-images/upload-images/custom-id.md:22]:
```suggestion
  -F 'id=<PATH_TO_YOUR_IMAGE.IMG>'
```

File: content/images/cloudflare-images/upload-images/custom-id.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+## Custom ID
+
+You can use a custom path to upload images to Cloudflare Images, instead of the path automatically generated by Cloudflare Images’ Universal Unique Identifier (UUID). The custom path has the following properties:
+
+- Could be up to 1024 characters long.
+- Could include any number of subpaths.
+- Requires file extensions. For example `.png` or `.jpg`.
+- Supports the [UTF-8 encoding standard](https://en.wikipedia.org/wiki/UTF-8) for characters.
+
+Below is an example of the custom ID feature:
```
Comment (sejoker) [inline content/images/cloudflare-images/upload-images/custom-id.md]:
```suggestion
Below is an example of the custom ID feature using upload via URL:
```

File: content/images/cloudflare-images/upload-images/custom-id.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+```bash
+curl -X POST https://api.cloudflare.com/client/v4/accounts/<​​ACCOUNT_ID>/images/v1 \
```
Comment (sejoker) [inline content/images/cloudflare-images/upload-images/custom-id.md]:
```suggestion
curl --request POST https://api.cloudflare.com/client/v4/accounts/<​​ACCOUNT_ID>/images/v1 \
```

File: content/images/cloudflare-images/upload-images/custom-id.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+  -H "Authorization: Bearer <API_TOKEN>" \
```
Comment (sejoker) [inline content/images/cloudflare-images/upload-images/custom-id.md]:
```suggestion
  --header 'Authorization: Bearer :token' \
```

File: content/images/cloudflare-images/upload-images/custom-id.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+  --form 'url=https://<REMOTE_PATH_TO_IMAGE>' \
+  --form 'id=<PATH_TO_YOUR_IMAGE.IMG>'
+```
```
Comment (sejoker) [inline content/images/cloudflare-images/upload-images/custom-id.md:23]: alternatively here is an example of the custom ID feature with direct file upload:
```
curl --request POST \
  https://api.cloudflare.com/client/v4/accounts/<ACCOUNT_ID>/images/v1 \
  --header 'Authorization: Bearer :token' \
  --form 'file=@./<PATH_TO_YOUR_IMAGE>' \
  --from 'id=<PATH_TO_YOUR_IMAGE.IMG>'
```

Comment (marciocloudflare) [reply content/images/cloudflare-images/upload-images/custom-id.md:23]: done

**Review by sejoker — APPROVED** [review]

**Pull request merged**